### ISSUE TITLE
Need a better documentation to build and run a development version of PXP

### ISSUE THREAD

**Issue by pierresh** [pxp-lang/pxp#50]
Hello,

This project sounds very exciting. In 2024, PHP does deserve its superset to override its old weaknesses!
I am pretty sure you could get support from many PHP developers but as this tooling is written in Rust (surely an excellent choice!), it might be a bit difficult for most of us to step in due to lack of knowledge of this language. So far, I cannot see any documentation about how to run PXP.

Thus, I think we need a better description of how we could:
- install dependencies (if any)
- build our own version of pxp
- test that built version of pxp 

Then it would be easier for us to experiment and contribute to this promising project :)

**Comment by ryangjchandler** [comment]
@pierresh Yep, you're absolutely right about documentation. It is on my todo list, but as you can imagine, that list is quite long 😄

**Comment by gseganzerla** [comment]
@ryangjchandler 

send me a e-mail

I dont know much about rust but I would love learn it to help with this project.
